Add OSPO docs and related policies (#6)
kommitters/kadena: README.md

# Kadena

> This SDK allows you to interact with the kadena blockchain to perform transactions, create, update and test smart contracts.

Reviewer: IMO, this description could be improved a little bit. 
```suggestion
This SDK allows you to interact with the Kadena blockchain to perform transactions, and create, update and test smart contracts.
```

## Installation

> The package can be installed
> by adding `kadena` to your list of dependencies in `mix.exs`:

Reviewer: What if we add a more direct message? 
```suggestion
Add `kadena` to your list of dependencies in `mix.exs`:
```

end
```

## Configuration
```elixir
config :kadena, network: :test # Default is `:test`. To use the public network, set it to `:public`

> 

Reviewer 2: We have not yet defined how to manage Kadena's network configuration. This is an assumption

> ```

Reviewer 2: @santiagoboterokommit why add this config without an implementation?

Author: The [next PR](https://github.com/kommitters/kadena/pull/7) will add a config folder with that lines :thinking:

Reviewer 2: @santiagoboterokommit it should be added in the next PR then
